safe-global/safe-react
# feat: notification centre

## What it solves
Resolves #3950

## How this PR fixes it
Notifications are now cached locally and are accessible via a notification centre. It is opened by clicking the bell in the header.

Notifications are unread if the user does not manually close the notification. Unread notifications are displayed at the top of the notification centre and marked as read when closing the notification centre if they were visible in the list.

Note: 'actionable' notifications are not implemented yet, hence why the "Required actions" section of the notification centre/notification actions are not present. These will be implemented as part of the https://github.com/safe-global/safe-react/issues/3951 epic.

## How to test it
Dispatch notifications via requesting a transaction signature, accessing a Safe that requires an update, etc. and observe that they are accessible in the notification centre.

## Screenshots
![notification centre](https://user-images.githubusercontent.com/20442784/173805974-0a4e9474-c3e9-450a-a0c5-a21a370d662b.gif)
